tests/examples/opinions/united_states/mass_example.compare.json

```diff
@@ -1,52 +1,446 @@
+    "docket_numbers": "SJC-13471",
+    "judges": "",
+    "summaries": "The respondent attorney, Edward J. Collins, appeals from the order of a single justice of this court suspending him from the practice of law. The matter came before the single justice on the information and record of proceedings filed by the Board of Bar Overseers (board). The board determined that in his representation of a particular client (client), the goal of which was to prevent her eviction, the respondent failed to act with diligence, consistently presented",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2024-09-18",
+    "case_names": "IN THE MATTER OF F.A.",
+    "download_urls": "https://www.socialaw.com/services/slip-opinions/slip-opinion-details/in-the-matter-of-f.a",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "SJC-13515",
+    "judges": "Budd, C.J., Gaziano, Kafker, Wendlandt, & Georges, JJ.",
+    "summaries": "Petition for civil commitment filed in the Worcester Division of the District Court Department on April 8, 2020. A motion for a buildings and grounds restriction was heard by Janet J. McGuiggan, J. The Supreme Judicial Court granted an application for direct appellate",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2024-09-17",
+    "case_names": "IN THE MATTER OF J.P. / IN THE MATTER OF E.S.",
+    "download_urls": "https://www.socialaw.com/services/slip-opinions/slip-opinion-details/in-the-matter-of-j.p.-in-the-matter-of-e.s",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "SJC-13492 / SJC-13493",
+    "judges": "Budd, C.J., Gaziano, Kafker, Wendlandt, & Georges, JJ.",
+    "summaries": "Petition for commitment for alcohol or substance use disorder filed in the Bristol County Division of the Juvenile Court Department on May 1, 2023. The case was heard by Michaela C. Stewart, J. The Supreme Judicial Court granted an application for direct appellate",
+    "case_name_shorts": ""
+  },
+  {
+    "case_dates": "2024-09-17",
+    "case_names": "COMMONWEALTH v. DANIEL ROGERS",
+    "download_urls": "https://www.socialaw.com/services/slip-opinions/slip-opinion-details/commonwealth-vs.-daniel-rogers",
+    "precedential_statuses": "Published",
+    "blocked_statuses": false,
+    "date_filed_is_approximate": false,
+    "docket_numbers": "SJC-13450",
+    "judges": "Budd, C.J., Kafker, Wendlandt, Georges, & Dewar, JJ.",
+    "summaries": "Indictments found and returned in the Superior Court Department on May 12, 2004. Following review by this court, 459 Mass. 249 (2011), a renewed motion to reduce the verdict and a motion for a new trial, filed on January 3, 2018, were heard by John A. Agostini, J. A request for leave to appeal was reported by Lowy, J., in the Supreme Judicial Court for the county of Suffolk, and a",
```

Comment:
a bunch of the summaries seem to be cut short in the middle of a sentence. Should we ingest these partial strings?

Reply:
I think so... let me check how common it is.

Reply:
yes - we should drop it ... they more often than not end early and I dont know why they did it